pkg/api/v1beta3/dynakube/logmonitoring/props.go

```diff
@@ -0,0 +1,21 @@
+package logmonitoring
+
+const (
+	logModuleDaemonSetSuffix = "-logmodule"
+)
+
+func (logMonitoring *Spec) SetName(name string) {
+	logMonitoring.name = name
+}
+
+func (logMonitoring *Spec) Needed() bool {
```

Comment:
```suggestion
func (logMonitoring *Spec) IsNeeded() bool {
```

Reply:
Don't have the strongest opinion for this one - I just think it fits the rest of our codebase better. You could also go with `Enabled()` but this is just my opinion, maybe you and/or the others are fine with `Needed()`